test/WebSites/BasicWebSite/Controllers/RemoteAttribute_HomeController.cs

```diff
@@ -19,23 +18,21 @@ public IActionResult Create()
         }
 
         [HttpPost]
-        public IActionResult Create(Person person)
+        public IActionResult Create(RemoteAttributeUser user)
         {
             ModelState.Remove("id");
             if (!ModelState.IsValid)
             {
-                return View(person);
+                return View(user);
             }
 
-            _person = person;
+            _user = user;
             return RedirectToAction(nameof(Details));
         }
 
-        [Route("/[Area]", Name = "AriaHome", Order = -3)]
-        [Route("/[Area]/[Controller]/Index", Order = -2)]
```

Comment:
Odd Details view isn't failing. Should have a route named `AriaHome` or update the `RouteLink()` call to use name `Area1Home` and restore at least one `[Route]` here.